kubetest/e2e.go

```diff
@@ -515,12 +515,20 @@ func isUp(d deployer) error {
 	return nil
 }
 
-func defaultDumpClusterLogs(localArtifactsDir, logexporterGCSPath string) error {
-	logDumpPath := "./cluster/log-dump/log-dump.sh"
-	// cluster/log-dump/log-dump.sh only exists in the Kubernetes tree
-	// post-1.3. If it doesn't exist, print a debug log but do not report an error.
+func logDumpPath(provider string) string {
+	// Use the log dumping script outside of kubernetes/kubernetes repo.
+	if os.Getenv("USE_KUBEKINS_LOG_DUMPING") != "" && (provider == "gce" || provider == "gke") {
```

Comment:
Please elaborate here why are you guarding against provider.

Reply:
Done.